packages/matchbox/src/components/Toggle/Toggle.js

```diff
@@ -42,7 +43,8 @@ function Toggle(props) {
       <StyledInput
         id={id}
         value={value}
-        defaultChecked={checked}
+        defaultChecked={defaultChecked}
```

Comment:
Nothing to fix now, but when fixing up the stories, it would be nice to have an example of how this works and it looks like the decorator is missing the StoryContainer.